### PR TITLE
Notebook about tesscut using requests

### PR DESCRIPTION
**Pull request by mustaric**
Please review this notebook. I would also appreciate suggestions on whether we need to educate more about the difference between astrocut and tesscut. Is that necessary. And did I get the terminology right in the notebook?

### CODE REVIEW

**Comment by mustaric** [comment]
Oh and remember this notebook can't really be approved until the data goes live because it uses dev.

**Comment by ceb8** [comment]
@mustaric  I added WIP to the ticket title to help us remember that it can't be merged yet (and why the tests are failing).

File: notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"catalog_ID\"></a>\n",
+    "## Get RA and Dec of your Target Using Astroquery Catalogs\n",
+    "\n",
+    "Here we use the astroquery.mast Catalogs query_criteria function to request the RA and Dec of the object with TIC ID = 261136679. This happens to be Pi Mensae. We collect the Right Ascension and Declination and store them in their own variables.  \n",
+    "<p>\n",
+    "Here we do a cone search using Catalogs.query_object on the TIC catalog. The advantage of doing this is that it gives us the nearby stars as well as the star we are looking for. The resulting table is sorted by distance from the requested object. We print out the ID and a few other TIC quantities to ensure we found the star we were looking for. "
```
Comment (ceb8) [inline notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb]: Suggest Catalogs.query_object -> \`Catalogs.query_object\` to set it off.

File: notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+    "ticid = 261136679\n",
+    "\n",
+    "starName = \"TIC \" + str(ticid)\n",
+    "rad_search = 5 / 60 #radius in degrees\n",
+    "\n",
+    "catalogData = Catalogs.query_object(starName, radius = rad_search, catalog = \"TIC\")\n",
+    "Ra = catalogData[0]['ra']\n",
+    "Dec = catalogData[0]['dec']\n",
+    "\n",
+    "#Print out the first row in the table\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb]: first row -> first 5 rows

Also, add a space after #.

File: notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Create a list of nearby bright stars (tess magnitude less than 14) from the rest of the data for later.\n",
+    "bright=catalogData['Tmag'] < 14\n",
```
Comment (ceb8) [inline notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb]: Add space after #, and around = . You've mostly done this throughout, but there are a few places that are still missing spaces.

File: notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"tesscut_ID\"></a>\n",
+    "## Request a FFI Cutout with Astrocut\n",
+    "Astrocut is the tool that runs the cutout service around the RA and Dec that were requested. It delivers a zipped file containing a cutout for each set of FFIs as listed above. It is also possible to request only one sector using the \"sector\" parameter.  For tesscut x refers to the CCD columns and y refers to the CCD rows. Distance can be input in a variety of units, I picked pixels (\"px\").\n",
+    "\n",
+    "<Response [200]> means that your request succeeded."
```
Comment (ceb8) [inline notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb]: Need to change <Response [200]> to < Response [200] > (spaced around Response [200]) or the whole things doesn't render.

**Comment by mustaric** [comment]
This one will stay in requests. I'm also building an astroquery one. 
Thanks for the comments.

**Comment by ceb8** [comment]
@mustaric  That makes sense. :)

File: notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb (hunk 4 above)
Comment (mustaric) [reply notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb]: I don't understand why the spaces are needed?  It renders just fine in my notebook and it more closely matches what is returned by the command.

Comment (ceb8) [reply notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb]: It renders in your notebook? Huh, in mine, without the spaces it's just... gone, like all I see is "means that your request succeeded." I'll see if updating jupyter does the trick.

Comment (mustaric) [reply notebooks/MAST/TESS/interm_tesscut_requests/interm_tesscut_requests.ipynb]: went ahead and changed it to spaces.

**Pull request merged**